meson.build

```diff
@@ -860,6 +860,11 @@ features += {'sndio': sndio.found()}
 if features['sndio']
     dependencies += sndio
     sources += files('audio/out/ao_sndio.c')
+
+    if sndio.version().version_compare('>=1.9.0')
+        flags += ['-DSNDIO_SIO_FLUSH']
+        add_project_arguments(flags, language: 'c')
+    endif
```

Comment:
Ah, the current convention is to just add it as a feature. For example, see the rubberband handling. So you could do something like `features += {'sndio-1-9': sndio.version().version_compare('>= 1.9.0')}` and that will create a HAVE_SNDIO_1_9 you can use.

Reply:
Ah, thanks. That is much simpler.